### PR TITLE
Create /repositories endpoints in swagger with handlers and use in UI

### PR DESCRIPTION
**Pull request by metalmatze**
This PR is converting all repositories related requests from GraphQL to RESTful Swagger based requests.

Much of the code is generated, but please review the following files:

[internal/api/v1/api.go](https://github.com/gitpods/gitpods/pull/25/files#diff-d5527cacb9976050ed370728596fd056)
[repository/service.go](https://github.com/gitpods/gitpods/pull/25/files#diff-736ba2f5bff9f02662eacaf17d59fe8a)
[repository/store_postgres.go](https://github.com/gitpods/gitpods/pull/25/files#diff-a6be1fe5f294323005885dca07aee112)
[repository/validate.go](https://github.com/gitpods/gitpods/pull/25/files#diff-9b506242afe39e7254ea90a010aa0d9a)
[repository/validate_test.go](https://github.com/gitpods/gitpods/pull/25/files#diff-ce8163cea6807f52e2f5c76976479c64)
[swagger.yaml](https://github.com/gitpods/gitpods/pull/25/files#diff-bac86ae86657c2fa2c3dd2ab26df4960)

Especially swagger.yaml :relaxed: 

Thanks.

/cc @bkcsoft @tboerger

### CODE REVIEW

File: repository/validate_test.go [hunk 1]
```diff
@@ -1,29 +1,93 @@
...
+		{
+			Name:       "NameTooLong",
+			Repository: &Repository{Name: "thisnameiswaytolongtobeadecentusername"},
+			Errors: []ValidationError{{
+				Field: "name",
+				Error: errors.New("name is not between 4 and 32 characters long"),
+			}},
+		},
+		{
+			Name:       "NameTooLong",
```
Comment (tboerger) [inline repository/validate_test.go]: Duplicate of the previous test case

File: swagger.yaml [hunk 2]
```diff
@@ -19,6 +19,98 @@ schemes:
...
+            type: object
+            required:
+              - name
+            properties:
+              name:
+                type: string
+              description:
+                type: string
+              website:
+                type: string
```
Comment (tboerger) [inline swagger.yaml]: Isn't there a url type?

Comment (metalmatze) [reply swagger.yaml]: Sadly there's not, but we check during validation with `govalidator.IsURL`

File: repository/validate_test.go (hunk 1 above)
Comment (metalmatze) [reply repository/validate_test.go]: Thanks. Removed it.

**Pull request merged**